tables/wordcx.dis

```diff
@@ -0,0 +1,90 @@
+#
+# Useful for convertion from or to the charset used to simulate
+# braille patterns on screen in Word CX / CX for Word. The table is 6
+# dot only. The character mapping is derived from the publication
+# Håndtok i punktskrift, Offentlig utvalg for punktskrift.
```

Comment:
Damn so this too late, but

1. this looks like a type: it's probably "håndbok"
2. would be nice to have an actual link to the document

Reply:
OK, fixed the typo and added a reference